lib/v2/screens/explore_screens/invite/interactor/mappers/create_invite_result_state_mapper.dart

```diff
@@ -0,0 +1,25 @@
+// import 'package:seeds/v2/datasource/remote/model/transaction_response.dart';
```

Comment:
do we need this ?

Reply:
I commented it, for the same reason of may question.